lib/src/layer/tile_layer/tile_provider/network_no_retry_image_provider.dart

```diff
@@ -29,7 +29,6 @@ class FMNetworkNoRetryImageProvider
     FMNetworkNoRetryImageProvider key,
     DecoderCallback decode,
   ) {
-    //ignore: close_sinks
```

Comment:
This line needs to be added back, as not having it is causing the checks to fail.

Reply:
`close_sinks` is already ignore in [analysis_options.yaml](https://github.com/fleaflet/flutter_map/blob/a6fb00f5887b6417d9bcfee8c407377618487906/analysis_options.yaml).

Reply:
Ok, in that case, can you stop ignoring it overall, because that's a useful lint, and just ignore it on this occasion?

Reply:
This lint is not in flutter_lints or custom add rules

Reply:
As below.

Reply:
readded